test/unit/test_devices/test_iosxewlc_device.py

```diff
@@ -0,0 +1,135 @@
+import mock
+import pytest
+
+from pyntc.devices import IOSXEWLCDevice
+from pyntc.devices import iosxewlc_device as iosxewlc_module
+
+
+def test_show(iosxewlc_send_command):
+    command = "show_ip_arp"
+    device = iosxewlc_send_command([f"{command}.txt"])
+    device.show(command)
+    device.native.send_command.assert_called_with(command_string="show_ip_arp")
+    device.native.send_command.assert_called_once()
```

Comment:
This should mock and assert on `IOSXEWLCDevice._send_command`

Reply:
Updated.